File: app/src/main/java/com/mdmobile/pocketconsole/gson/DeviceUserInfo.java
package com.mdmobile.pocketconsole.gson;

/**
 * DeviceUserInfo class -> present in all devices Kind beside Zebra printers
 */

public class DeviceUserInfo {

        private String UserName, FirstName, MiddleName, LastName, DomainName, UPN, PhoneNumber,
                Email, CustomProperty1, CustomProperty2, CustomProperty3, Identifier;

        public String getUserName() {
            return UserName;
        }

        public String getFirstName() {
            return FirstName;
        }

        public String getMiddleName() {
            return MiddleName;
        }

        public String getLastName() {
            return LastName;
        }

        public String getDomainName() {
            return DomainName;
        }

        public String getUPN() {
            return UPN;
        }

        public String getPhoneNumber() {
            return PhoneNumber;
        }

        public String getEmail() {
            return Email;
        }

        public String getCustomProperty1() {
            return CustomProperty1;
        }

        public String getCustomProperty2() {
            return CustomProperty2;
        }

        public String getCustomProperty3() {
            return CustomProperty3;
        }

        public String getIdentifier() {
            return Identifier;
        }
}

File: app/src/main/java/com/mdmobile/cyclops/dataModel/api/devices/test/DeviceUserInfo.kt
package com.mdmobile.cyclops.dataModel.api.devices.test

import com.google.gson.annotations.SerializedName

data class DeviceUserInfo(
        @SerializedName("UserName") val userName: Any = Any(),
        @SerializedName("FirstName") val firstName: Any = Any(),
        @SerializedName("MiddleName") val middleName: Any = Any(),
        @SerializedName("LastName") val lastName: Any = Any(),
        @SerializedName("DomainName") val domainName: Any = Any(),
        @SerializedName("UPN") val uPN: Any = Any(),
        @SerializedName("PhoneNumber") val phoneNumber: Any = Any(),
        @SerializedName("Email") val email: Any = Any(),
        @SerializedName("CustomProperty1") val customProperty1: Any = Any(),
        @SerializedName("CustomProperty2") val customProperty2: Any = Any(),
        @SerializedName("CustomProperty3") val customProperty3: Any = Any(),
        @SerializedName("Identifier") val identifier: Any = Any()
)